core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -86,8 +85,14 @@ def _migrate_exploration(
             with datastore_services.get_ndb_context():
                 if exp_services.get_story_id_linked_to_exploration(
                         exp_id) is not None:
-                    exp_services.validate_exploration_for_story(
-                        exploration, True)
+                    try:
+                        exp_services.validate_exploration_for_story(
+                            exploration, True)
+                    except exp_services.UnsupportedRTEContentValidationError:
+                        # The unsupported RTE content error is supressed as we
+                        # need to fix the error in existing exploration before
+                        # enabling this check.
+                        pass
```

Comment:
I thought Sean is supposed to fix this before we merge this PR?

Reply:
Some explorations are not in a state to edit and they also contain these errors so we are going to migrate them and once it's migrated we can edit it and fix the issues.

Reply:
Hmm, from the testing of Hiteshe's job I think we got rid of all the issues even without this hack or maybe I'm just not sure what is this supposed to do.

Reply:
Hitesh's job doesn't covers all the explorations ([ref](https://github.com/oppia/oppia/blob/82373416488e46c3b93887b22543cff9dbbb75b3/core/jobs/batch_jobs/exp_migration_jobs.py#L606-L608)), the uncovered explorations have these issues and we cannot edit them without this new migration!

Reply:
Oh, makes sense. Can we add a TODO for removing this after the migration into the code?

Reply:
I've removed the UnsupportedRTEContentValidationError now, as per our discussion in another thread! (This is not needed anymore)